CHANGELOG.md

```diff
@@ -7,6 +7,43 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 <!-- towncrier release notes -->
 
+## [v0.96.0-sumo-0]
+
+### Released 2024-03-07
+
+### Breaking Changes
+
+- feat(sumologicexporter)!: remove deprecated json_logs [#1452]
+- feat(sumologicexporter)!: remove deprecated clear_logs_timestamp [#1455]
+
+### Added
+
+- ci: build ubi-based images [#1440]
+- feat(k8sprocessor): extract namespace annotations [#1457]
+- ci: build UBI-based FIPS images [#1463]
+- feat: build windows containers [#1467]
```

Comment:
I think we're only building these on `main`, so this in the changelog might confuse users. WDYT @sumo-drosiek ?

Reply:
If a feature is not available in the release builds, we should either not mention it at all in the README, or explicitly call out that (like in this case) it is only available for development builds.

Reply:
I change this changelog enty, I hope that now it is clear